core/store/src/db/refcount.rs

```diff
@@ -8,14 +8,15 @@
 //!
 //! The reference counts are stored together with the values by simply attaching
 //! little-endian encoded 64-bit signed integer at the end of it.  See
-//! [`encode_value_with_rc`] for more details about the encoding.  During
-//! compaction, RocksDB merges the values by adding the reference counts.  When
-//! the reference count reaches zero RocksDB removes the key from the database.
+//! [`add_positive_refcount`] and [`encode_negative_refcount`] functions for
+//! a way to encode reference count.  During compaction, RocksDB merges the
+//! values by adding the reference counts.  When the reference count reaches
+//! zero RocksDB removes the key from the database.
 
 use std::cmp::Ordering;
-use std::io::{Cursor, Write};
+use std::io::Cursor;
 
-use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
+use byteorder::{LittleEndian, ReadBytesExt};
```

Comment:
Maybe replace decoding with `from_le_bytes` if we are changing encode path anyway?

Reply:
I have another commit incoming which will rework merge_refcounted_records.